bin/compile

```diff
@@ -13,6 +13,14 @@ unset GIT_DIR
 BUILD_DIR=$1
 BUILDPACK_DIR="$(dirname $(dirname $0))"
 
+STUNNEL=$(command -v stunnel4)
+
+if [ -z $STUNNEL ]; then
+    echo "------> stunnel not detected! stunnel not supported on heroku-24+"
```

Comment:
Could you add a mention of the buildpack deprecation/sunset/not supporting on Heroku-24 to the README?

Reply:
Absolutely.